examples/java/src/main/java/org/apache/beam/examples/common/ExampleUtils.java

```diff
@@ -79,7 +80,9 @@ public ExampleUtils(PipelineOptions options) {
    */
   public void setup() throws IOException {
     Sleeper sleeper = Sleeper.DEFAULT;
-    BackOff backOff = new AttemptBoundedExponentialBackOff(3, 200);
+    BackOff backOff =
+        FlexibleBackoff.of()
```

Comment:
`FluentBackoff` (as analog to FluentIterable, etc.)

I'd also consider getting rid of the `of()` static factory, and instead just requiring the methods to configure it to be called.